src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side) {
```

Comment:
Add a constructor that takes both side length and color. This allows creating Square objects with a specific color. Example: `public Square(int side, String color) { ... }` 🎨